examples/shell_automatic_cd.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
Let's remove the shebang and mention this script is supposed to be sourced. You can't define a function by running a script anyway.

Reply:
Done.

Reply:
Could you add explanation that these files are supposed to be sourced from .bashrc/.zshrc/etc.?

Reply:
Sure, just did that.

Reply:
Great, but it's still missing from `shell_subshell_notice.sh`.

Reply:
That file already contains the following line:

```
# Add this line to your shell startup file (.bashrc, .zshrc etc) for it to work.
```

Reply:
Yeah but it'd be smarter to source the file from the repo in your `.bashrc`, that way you'd get updates if we ever change it.

Reply:
Done.

BTW, just recalled why I added a shebang to these two files: I did that to specify the type of shell for `shellcheck`.

Reply:
Let's do that this way instead, because we don't want to suggest running the scripts:

Reply:
Wow, didn't know that `shellcheck` had directives (especially this one)! Thanks a lot!